db/orders.js

```diff
@@ -0,0 +1,94 @@
+const client = require("./client")
```

Comment:
I came up with a way we might restructure the orders in the database to be more like it was with routines and activities in fitnesstrackr that I think makes some api stuff easier/less time complexity.

Reply:
Finished changing order_items to orders and created the new order_puppies table.